neqo-transport/src/path.rs

```diff
@@ -1009,9 +1009,11 @@ impl Path {
 
         let ecn_ce_received = self.ecn_info.on_packets_acked(acked_pkts, ack_ecn, stats);
         if ecn_ce_received {
-            let cwnd_reduced = self
-                .sender
-                .on_ecn_ce_received(acked_pkts.first().expect("must be there"), now);
+            let cwnd_reduced = self.sender.on_ecn_ce_received(
+                acked_pkts.first().expect("must be there"),
```

Comment:
Is this `expect` safe?

Reply:
Yes it is safe. We never call `on_packets_acked` with an empty list of acked packets.

https://github.com/mozilla/neqo/blob/4aebca85325bcb8c98ee16480c3cc263d18c606c/neqo-transport/src/recovery/mod.rs#L639-L642

That said, the `expect` being in a different file than the code upholding this invariant is less than ideal.